OsmAnd/src/net/osmand/plus/plugins/development/OsmandDevelopmentPlugin.java

```diff
@@ -262,4 +279,89 @@ public OrderedLineDataSet getOrderedLineDataSet(@NonNull LineChart chart, @NonNu
 	protected TrackPointsAnalyser getTrackPointsAnalyser() {
 		return AutoZoomBySpeedHelper.getTrackPointsAnalyser(app);
 	}
-}
\ No newline at end of file
+
+	private boolean fpsStatsEnabled = false;
+	private final int FPS_STATS_INTERVAL_SECONDS = 1; // 10 ?
+	private final int FPS_STATS_LIFETIME_MINUTES = 15;
+	private Handler fpsStatsHandler = new Handler(Looper.getMainLooper());
+	private List<FpsStatsEntry> fpsStats = Collections.synchronizedList(new ArrayList<>());
```

Comment:
do not use Collections.synchronizedList, recreate list when changed

Reply:
cow style done